### PR TITLE
fix memory limit for cgroups2 #1782

### PR DESCRIPTION
**Pull request by rigazilla**
proposal solution for  #1782

### CODE REVIEW

File: pkg/kubernetes/pods.go [hunk 1]
```diff
@@ -156,7 +156,7 @@ func GetContainer(name string, spec *corev1.PodSpec) *corev1.Container {
 func GetPodMemoryLimitBytes(container, podName, namespace string, kube *Kubernetes) (uint64, error) {
 	execOut, err := kube.ExecWithOptions(ExecOptions{
 		Container: container,
-		Command:   []string{"cat", "/sys/fs/cgroup/memory/memory.limit_in_bytes"},
+		Command:   []string{"/bin/bash", "-c", "cat /sys/fs/cgroup/memory/memory.limit_in_bytes || cat /sys/fs/cgroup/memory.max"},
```
Comment (ryanemerson) [inline pkg/kubernetes/pods.go:159]: Won't this result in the output containing something like:

```
cat: /sys/fs/cgroup/memory/memory.limit_in_bytes :No such file or directory
```

Followed by the value in the second file?

Comment (rigazilla) [reply pkg/kubernetes/pods.go:159]: mmm probably yes... It worked locally but let me check

Comment (rigazilla) [reply pkg/kubernetes/pods.go:159]: No, it works, only the stdout is returned, stderr if error happens is wrapper in an error instance:
https://github.com/infinispan/infinispan-operator/blob/bdfc2db793ee21e33f89a0d99be491d7c0a95bf6/pkg/kubernetes/kubernetes.go#L150-L159

Comment (ryanemerson) [reply pkg/kubernetes/pods.go:159]: O nice, thanks for double checking :+1:

**Comment by ryanemerson** [comment]
Thanks @rigazilla

**Pull request merged**